code/modules/surgery/revival.dm

```diff
@@ -57,6 +57,8 @@
 		"<span class='notice'>[user] prepares to shock [target]'s brain with [tool].</span>",
 		"<span class='notice'>[user] prepares to shock [target]'s brain with [tool].</span>")
 	target.notify_ghost_cloning("Someone is trying to zap your brain. Re-enter your corpse if you want to be revived!", source = target)
+	if(HAS_TRAIT(target,TRAIT_SURGERYPLUS))
```

Comment:
this will very, very rarely occur, as medibots don't target dead people, right?

Reply:
correct